client/src/components/EventEdit/Header/index.js

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import Moment from 'react-moment';
+import PropTypes from "prop-types";
+import { FiPrinter, FiSettings, FiDownload } from "react-icons/fi";
+import styled from "styled-components";
+
+import IconButton from "../../Common/IconButton";
+import {
+  theme1,
+  fontSize64,
+  fontSize24,
+  spacing24,
+} from "../../../resources/style-constants";
+
+const Container = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  background-color: ${props => props.bgcolor || theme1};
+`;
+
+const Spacer = styled.div`
+  height: ${spacing24};
+`;
+
+const TitleContainer = styled.h1`
+  font-size: ${fontSize64};
+  margin: 0;
+`;
+
+const DateContainer = styled.h5`
+  font-size: ${fontSize24};
+  opacity: 0.5;
+  margin: 0;
+`;
+
+const ButtonBarContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+/**
+ * The header which appears at the top of the event-edit page.
+ */
+const Header = ({
+  title,
+  startDate,
```

Comment:
One idea for the dates is we can have them select an actual start date (so we can properly sort the different events) and then we could have a text input called "Display Date". Then the user can decide exactly how they want to display the date and time of their event. We can address this when we work on editing the title and date.

Reply:
I like this idea. This simplifies the date issue a lot, good thinking